--- download_prerequisites	2015-12-22 18:08:55.712765232 +0100
+++ new_down	2015-12-22 18:02:41.511459121 +0100
@@ -25,1 +25,1 @@
 GRAPHITE_LOOP_OPT=yes

 # Necessary to build GCC.
-MPFR=mpfr-2.4.2
-GMP=gmp-4.3.2
-MPC=mpc-0.8.1
+MPFR=mpfr-3.1.4
+GMP=gmp-6.1.0
+MPC=mpc-1.0.3

-wget ftp://gcc.gnu.org/pub/gcc/infrastructure/$MPFR.tar.bz2 || exit 1
+wget http://www.mpfr.org/mpfr-current/${MPFR}.tar.bz2 || exit 1
 tar xjf $MPFR.tar.bz2 || exit 1
 ln -sf $MPFR mpfr || exit 1

-wget ftp://gcc.gnu.org/pub/gcc/infrastructure/$GMP.tar.bz2 || exit 1
+wget https://gmplib.org/download/gmp/${GMP}.tar.bz2 || exit 1
 tar xjf $GMP.tar.bz2  || exit 1
 ln -sf $GMP gmp || exit 1

-wget ftp://gcc.gnu.org/pub/gcc/infrastructure/$MPC.tar.gz || exit 1
+wget ftp://ftp.gnu.org/gnu/mpc/${MPC}.tar.gz || exit 1
 tar xzf $MPC.tar.gz || exit 1
 ln -sf $MPC mpc || exit 1
